include/abb_librws/v1_0/rws_progress.h

```diff
@@ -13,13 +13,16 @@ namespace abb :: rws :: v1_0 :: progress
     };
 
     /**
-   * \brief Checks progress of provided resource.
-   * 
-   * \param id Resource id to check
-   *
-   * \return \a ProgressInfo containing the result.
-   *
-   * \throw \a RWSError if something goes wrong.
-   */
-    ProgressInfo getProgress(RWSClient& client, std::string const& id);
-}
\ No newline at end of file
+     * \brief Checks progress of provided resource.
+     *
+     * \param id Resource id to check
+     *
+     * \return \a ProgressInfo containing the result.
+     *
+     * \throw \a RWSError if something goes wrong.
+     */
+    ProgressInfo getProgress(RWSClient &client, std::string const &id);
+
+    const std::string READY = "ready";
+    const std::string PENDING = "pending";
```

Comment:
I also suggest moving these constants to a struct, s.t. they syntactically look like an enum class. First, it makes clear that these constants define progress status. Second, if at some point we decide to use an enum for progress status instead of a string, the syntax on the client side will not change.
```suggestion
    struct ProgressStatus final
    {
        static const std::string READY;
        static const std::string PENDING;
    };
```